### DISCUSSION TITLE
Question about Add-on development for vanilla clients

### DISCUSSION THREAD

**Discussion by AnthonyPaulO** [chromiecraft/chromiecraft#5219] [General]
Hey guys,

I've just started learning add-on development for wow and was wondering if anyone knows of a repo that contains the client DBCs vanilla, tbc-vanilla, and wotlk-vanilla. The only one I've seen, which is from Questie's github repo, has issues.

Thanks!

Anthony

**Comment by r-o-b-o-t-o** [comment]
[wow.tools](https://wow.tools/dbc/) is great, you can browse the DBCs for most client patches.

Otherwise you can use Ladik's MPQ Editor to extract the contents of the MPQ files from the client and browse the DBCs' data with [WDBXEditor](https://github.com/WowDevTools/WDBXEditor)

**Reply by AnthonyPaulO** [reply]
Thank you so much, this seems to be exactly what I need! FYI, WoWTools data isn't perfect, has some issues, which is why I'm asking.
